.circleci/config.yml

```diff
@@ -70,6 +70,15 @@ commands:
       - run:
           name: pip install
           command: << parameters.python >> -m pip install -r requirements-dev.txt
+  install-rust:
+    steps:
+      - run:
+          name: install rust
+          command: |
+            curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
#YOLO

Reply:
This is the official way to install rust apparently.. so its actually good that we use this method here I think.